pom.xml

```diff
@@ -3,9 +3,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.30</version>
```

Comment:
Not strictly related, but 2.30 is now 7 months outdated: https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md

Reply:
Would prefer to do upgrades in a separate PR. I just wanted to get it working with this.